examples/fit_with_bounds.py

```diff
@@ -19,12 +19,8 @@
 np.random.seed(15816)
 # {{{ helper function(s)
 class myfitclass (object):
-    def this_expression(self):
-        A, shift, period, decay, x = sp.symbols("A shift period decay x")
-        expr = A * sp.sin(shift + x / period) * sp.exp(-((x * decay) ** 2))
-        return expr
-
-    def gen_from_expr(self, expr, data, guesses={}):
+    expression = (2+2)
```

Comment:
you don't want to initialize the expression to something random.

There are different ways of approaching this, but I would go like this:
```
def __init__():
    self.expression = None
```
then below, in `gen_from_expr`, you would add a line at the beginning:
```
    if self.expression is None: raise ValueError("Ahhh! But you haven't set the expression yet!!")
```

Reply:
okay this makes sense! will do!